lib/clueFromPosition.ts

```diff
@@ -28,7 +28,7 @@ export const clueInfosForPosition = (tiles: CrosswordJSON["tiles"], clues: Cross
 
 export const tilePositionsForClue = (clue: Clue, direction: CursorDirection): Position[] => {
   const start = clue.position
-  return Array.from({ length: clue.answer.length }).map((_, i) => {
+  return Array.from({ length: clue.tiles.length }).map((_, i) => {
```

Comment:
We can't rely on answer ever for navigation since it will have more characters than there are cells in rebus puzzles